README.md

```diff
@@ -96,11 +97,21 @@ If you need to drop all tables due to a schema change or any other reason, execu
 
 * Clone this repository to your computer
 * Run `npm install` in the newly-cloned directory
-* Run `npm run dev`
+* Run `npm run dev**
```

Comment:
this is broken

Reply:
Ya, thanks....... Not sure how I managed to pull that off.